exchangelib/account.py

```diff
@@ -119,8 +119,8 @@ def folders(self):
             folders_map[f.__class__].append(f)
         return folders_map
 
-    def sync_folder_hierarchy(self, shape, sync_state=None):
-        return SyncFolderHierarchy(account=self).call(shape, sync_state)
+    def sync_folder_hierarchy(self, shape, sync_state=None, additional_fields=None):
+        return SyncFolderHierarchy(account=self).call(shape, sync_state, additional_fields)
```

Comment:
Do we have to do any verification of inputs that it is in fact an additional field that exists on the Folder?

Reply:
We don't /have/ to. Do you think we should? We'd have to handle both `Field` and `FieldPath`, I'd have to do some digging to see if it's possible to tell if those relate back to `Folder`. Also exchangelib doesn't really do that type of verification on any of the other services that support additional fields

Reply:
Ok that makes sense. Lets leave it without verification.

Reply:
:+1: